src/common/ip-api.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Key for our pro subscription of ip-api (https://ip-api.com/) - used for
+ * geolocating and pre-populating user's locations on the homepage.
+ *
+ * Keeping the key here is fine (origin restrictions make it usable only for CAN)
```

Comment:
Maybe add a note saying that it won't work locally or in preview links so we don't forget about that